editor.planx.uk/src/@planx/components/PlanningConstraints/Public.test.tsx

```diff
@@ -22,27 +17,16 @@ jest.spyOn(SWR, "default").mockImplementation((url: any) => {
   } as any;
 });
 
-jest.spyOn(ReactNavi, "useCurrentRoute").mockImplementation(
-  () =>
-    ({
-      data: {
-        team: TEAM,
-      },
-    } as any)
-);
-
 it("renders correctly", async () => {
   const handleSubmit = jest.fn();
 
   const { user } = setup(
-    <MockedProvider mocks={teamMock} addTypename={false}>
-      <PlanningConstraints
-        title="Planning constraints"
-        description="Things that might affect your project"
-        fn="property.constraints.planning"
-        handleSubmit={handleSubmit}
-      />
-    </MockedProvider>
+    <PlanningConstraints
+      title="Planning constraints"
+      description="Things that might affect your project"
+      fn="property.constraints.planning"
+      handleSubmit={handleSubmit}
+    />
```

Comment:
:scissors: this diff is such a nice demo of changes / how much these store methods simplify !